core/uemis.c

```diff
@@ -374,8 +373,8 @@ void uemis_parse_divelog_binary(char *base64, void *datap)
 	/* get data from the footer */
 	char buffer[24];
 
-	snprintf(version, sizeof(version), "%1u.%02u", data[18], data[17]);
-	add_extra_data(dc, "FW Version", version);
+	snprintf(buffer, sizeof(buffer), "%1u.%02u", data[18], data[17]);
```

Comment:
The format string `%1u` is strange - isn't this equivalent to `%u`?

Likewise, two occurrences of `%0u` later in the same file could be written as `%u`, if I'm not mistaken.